common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -366,4 +381,45 @@ private boolean getMultipleCloudsSupported(@NonNull final OperationParameters pa
             return false;
         }
     }
+
+    /**
+     * List of System Browsers which can be used from broker, currently only Chrome is supported.
+     * This information here is populated from the default browser safelist in MSAL.
+     * @return
+     */
+    public static List<BrowserDescriptor> getBrowserSafeListForBroker(){
```

Comment:
For a future release we might consider how/what we might refactor here to not duplicate constants across common and msal. Maybe this just means moving the trusted hashes to `common`.....